src/js/components/DataTable/doc.js

```diff
@@ -347,6 +347,11 @@ export const themeDoc = {
     type: 'string | object',
     defaultValue: undefined,
   },
+  'dataTable.header.color': {
+    description: 'The text color in a header cell.',
```

Comment:
My understanding from the code is that it impacts the icon color rather than the text color, what am I missing?

Reply:
It affects both the text and the icon. Last night I referenced how button is documented https://v2.grommet.io/button#button.color to keep it consistent with that, however I think we should describe it as "The color of the label and icon in a header cell."

Reply:
Updated to read "The label and icon color in a header cell."